### PR TITLE
Create migration to fix zip3 923 issue

### PR DESCRIPTION
**Pull request by kimallen**
## Description

In dumping the `tariff400ng_zip3s` and `tariff400ng_zip5_rate_areas` data, I discovered that we were not indicating that zip3 923 had zip5-specific rate areas.  This updates the `rate_area` for the zip3 of 923 to from `US88` to `ZIP`, which indicates rate areas are found in the `tariff400ng_zip5_rate_areas` table.

## Code Review Verification Steps

* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/tree/master/docs/database.md#zero-downtime-migrations))
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-1282) for this change
[Slack convo](https://ustcdp3.slack.com/archives/CP6F568DC/p1579805982006100)

### CODE REVIEW

File: migrations/20200124192943_update_tariff400ng_mistaken_rate_area.up.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+UPDATE dev_db.public.tariff400ng_zip3s
```
Comment (reggieriser) [inline migrations/20200124192943_update_tariff400ng_mistaken_rate_area.up.sql]: Don't think you mean to put in `dev_db` here.

Comment (kimallen) [reply migrations/20200124192943_update_tariff400ng_mistaken_rate_area.up.sql]: Weird, that must have auto-populated

Comment (kimallen) [reply migrations/20200124192943_update_tariff400ng_mistaken_rate_area.up.sql]: fixed

**Comment by kimallen** [comment]
@lynzt , I'm not sure how much we do use the zip3 lookups.  We use them for the 'old' rate engine being used for calculating PPM incentives.  We are building out a different schema for the zip lookups for the GHC rate engine, and possibly we would want to think about indexing there.

**Pull request merged**